lib/splitclient-rb/version.rb

```diff
@@ -1,3 +1,3 @@
 module SplitIoClient
-  VERSION = '3.2.3'
+  VERSION = '3.2.4'
```

Comment:
@maksimf max please drop this change from here so it does not override of conflict with latest head.